app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/ColorCorrect.scala

```diff
@@ -211,12 +251,13 @@ object ColorCorrect {
     // Sequence of transformations to tile, flatten removes None from the list
     val transformations: List[MultibandTile => MultibandTile] = List(
       maybeEqualize,
-      Some(normalizeAndClampValues),
+      Some(normalizeLayer),
       maybeAdjustBrightness,
       maybeAdjustGamma,
       maybeAdjustContrast,
       maybeAdjustSaturation,
-      maybeSigmoidal
+      maybeSigmoidal,
+      Some(applyColorCorrections)
```

Comment:
Yeah I can revise this, it isn't wrapping the operation in an `Option` so much as removing the `getOrElse` from them so the return type is `Option[MultibandTile]` like the others.